org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/ProcessInfoDialog.java

```diff
@@ -216,6 +237,64 @@ public void onEvent(Event event) throws Exception {
 		if (event.getTarget() == btnOk) {
 			this.detach();
 		}
+		if(event.getTarget() == btnPrint) {
+			Clients.showBusy(Msg.getMsg(Env.getCtx(), "Processing"));
+			Executions.schedule(this.getDesktop(), e -> onPrint(), new Event("onPrint"));
+		}
+	}
+	
+	/**
+	 * On Print
+	 */
+	private void onPrint() {
+		// Loop through all items

```

Comment:
hmm ... sorry for the mistake in previous review, Clients.clearBusy(); should be the first line in this method, not the 2 place below. No matter what happens below, we need to make sure the busy mask is clear.